docs/reference/injector.md

```diff
@@ -0,0 +1,173 @@
+# Sidecar installation
+
+In order to use AWS App Mesh in Kubernetes, pods in the mesh must be running the AWS App Mesh sidecar proxy (Envoy).
+
+The following sections describe methods for injecting sidecar/Envoy into a pod for Virtual Nodes and Virtual Gateways
+
+## Envoy injection for virtual nodes
+
+Sidecars can be automatically added to Kubernetes pods using a [mutating webhook admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/) as part of the App Mesh Kubernetes controller.
+
+App Mesh uses namespace and/or pod annotations to determine if pods in a namespace will be marked for sidecar injection. There are two ways to achieve this:
+
+`appmesh.k8s.aws/sidecarInjectorWebhook: enabled`: The sidecar injector will inject the sidecar into pods by default. Add the `appmesh.k8s.aws/sidecarInjectorWebhook` annotation with value `disabled` to the pod template spec to override the default and disable injection. For example:
+
+```
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: default-disabled
```

Comment:
Should this be 'default-enabled'? I see the below deployment is tied to 'default-enabled'.

Reply:
Yess!! let me fix this